### PR TITLE
fix(Wizard): add wizard demos to match core

### LINKED ISSUE #4780: Create modal and in page Wizard demos for React
Currently there are no React demos for a Wizard. To have parity with Core, want to create interactive demos for a modal wizard and an in-page wizard.

### PR DESCRIPTION
**Pull request by nicolethoen**
Closes #4780

### CODE REVIEW

**Review by kmcfaul — APPROVED** [review]
lgtm

**Review by mcarrano — COMMENTED** [review]
This looks good @nicolethoen . I just have one question. For the In Page version. I notice that in the core demo, the buttons are anchored to the bottom of the page where in the react version they fall in the middle of the page. I don't have a strong opinion on which is correct, but unclear why they are different. Do we set a fixed height on the wizard or is it designed to fill the page? @mcoker thoughts?

File: packages/react-core/src/demos/Wizard/WizardDemo.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+                A demo of the wizard in page.
+              </Text>
+            </TextContent>
+          </PageSection>
+          <PageSection type={PageSectionTypes.wizard} variant={PageSectionVariants.light}>
+            <Wizard
+              navAriaLabel={`${title} steps`}
+              mainAriaLabel={`${title} content`}
+              steps={steps}
+              height={400}
```
Comment (mcoker) [inline packages/react-core/src/demos/Wizard/WizardDemo.md:219]:
```suggestion
```

**Review by mcoker — CHANGES_REQUESTED** [review]
Looks like there is an unnecessary height limitation on the in page wizard. That's what's keeping it from consuming the page height @mcarrano

**Review by jessiehuff — APPROVED** [review]
LGTM!

**Review by mcarrano — APPROVED** [review]
Looks great @nicolethoen .

**Review by mcoker — APPROVED** [review]
🤩👍

**Review by tlabaj — APPROVED** [review]

**Pull request merged**